src/haddock/modules/analysis/seletopclusts/defaults.cfg

```diff
@@ -1,2 +1,2 @@
-top_cluster = [1, 2, 3, 4]
-top_models = 4
+top_cluster = []
```

Comment:
Do we really want the defaults to provide all models? Or should the defaults have a more natural value?